packages/client/src/__tests__/integration/happy/chmod/test.ts

```diff
@@ -4,8 +4,10 @@ import fs from 'fs'
 import path from 'path'
 import { generateTestClient } from '../../../../utils/getTestClient'
 
+const testIf = (condition: boolean) => (condition ? test : test.skip)
+
 // Tests that no error is being thrown when the binary is manually set to chmod 644 because Client fixes that itself
-test('chmod', async () => {
+testIf(process.platform !== 'win32')('chmod', async () => {
```

Comment:
Does this fail on Windows otherwise?

Reply:
It failed for unrelated reason (which I can fix just to see what happens, but I didn't try that). I disabled the test instead because it doesn't make much sense on Windows, where there's no executable flag.

Also, the functionality that is being tested here is a no-op on Windows: https://github.com/prisma/prisma/blob/main/packages/engine-core/src/binary/BinaryEngine.ts#L437-L439